x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/normalization/rule_converters.ts

```diff
@@ -401,6 +400,11 @@ export const convertPatchAPIToInternalSchema = (
 ): InternalRuleUpdate => {
   const typeSpecificParams = patchTypeSpecificSnakeToCamel(nextParams, existingRule.params);
   const existingParams = existingRule.params;
+
+  const alertActions = nextParams.actions?.map(transformRuleToAlertAction) ?? existingRule.actions;
+  const throttle = nextParams.throttle ?? transformFromAlertThrottle(existingRule, null);
```

Comment:
nit: It's an interesting choice of providing `null` as `throttle` value instead of `undefined` though basically both `null` and `undefined` mean absence of a value. I try to extrapolate of using both later on and it can be confusing. Can we stick to `undefined` as minimum inside `Security Solution`?

There is a [task](https://github.com/elastic/kibana/issues/147736) I've been working on which requires removing of throttle normalization. It will lead it to fallback to `undefined` when it's not set as we don't use both `null` and `undefined` in our APIs according to the [schemas](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/detection_engine/rule_schema/model/rule_schemas.ts).

Reply:
This is gone after the Vitalii's changes been merged.